appinventor/appengine/src/com/google/appinventor/client/OdeMessages.java

```diff
@@ -5788,4 +5788,8 @@ String newerVersionComponentException(String componentType, int srcCompVersion,
   @DefaultMessage("Time")
   @Description("Set x-axis label values Type as Time")
   String labelTime();
+
+  @DefaultMessage("Welcome to App Inventor Neo! If you are looking for the classic App Inventor look, you can switch in the User Interface Settings, or <a href=\"\">click here</a>.")
```

Comment:
Should we be concerned about translations, or is this popup largely targeting English-speakers?

Reply:
We'll obviously want to update the translation files, and hopefully volunteers will translate it if they think it is a high priority. However, I don't think the lack of translations should halt merging it.